testing/yubico-piv-tool/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: Christophe BERAUD-DUFOUR <christophe.berauddufour@gmail.com>
+# Maintainer:
+pkgname=yubico-piv-tool
+pkgver=1.4.3
+pkgrel=0
+pkgdesc="PIV Tools for yubikey on RaspberryPi"
+url="https://developers.yubico.com/yubico-piv-tool"
+arch="all"
+license="BSD"
+depends="openssl acf-openssl libcrypto1.0 libressl2.5-libcrypto pcsc-lite ccid"
+makedepends="libc-dev gcc libressl-dev libcrypto1.0 libressl2.5-libcrypto pcsc-lite-dev"
+install=""
+subpackages="$pkgname-doc"
+source="https://developers.yubico.com/yubico-piv-tool/Releases/yubico-piv-tool-1.4.3.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+prepare() {
+	default_prepare || return
```

Comment:
The `prepare()` block is not needed. If it's not defined, `default_prepare` is done automatically. Also `|| return` is now redundant as abuild runs everything with `set -e`.